### DISCUSSION TITLE
v3.5.4 UI not loading consistently

### DISCUSSION THREAD

**Discussion by rkunalan** [argoproj/argo-workflows#13176] [Q&A] [answered]
We recently noticed our Argo UI sometimes not loading consistently (It started after we updated the Argo into recent versions). We verified that, we didn’t see any issues with the Argo server or controller manager. Most of the times a hard refresh fixed it. The page just continued to try to load the content. The main UI wrapper is present - just not the content its loading. It is mostly common on all the browsers but more frequent in Chrome. We are currently in v3.5.4. Any thoughts/suggestions would be helpful. Thanks.

**Comment by agilgur5** [comment, marked as answer]
> We are currently in v3.5.4

I would recommend updating to the latest patch, 3.5.7 at time of writing.

> It is mostly common on all the browsers but more frequent in Chrome.

This specifically sounds like a duplicate of #12626, which was fixed in a newer patch.
